module/VuFind/src/VuFind/ILS/Driver/KohaILSDI.php

```diff
@@ -604,6 +604,22 @@ public function placeHold($holdDetails)
         $this->debug("Needed before date: " . $needed_before_date);
         $this->debug("Level: " . $level);
 
+        // The following check is mainly required for certain old buggy Koha versions
+        // that allowed multiple holds from the same user to the same item
+        $sql = "select count(*) as RCOUNT from reserves where borrowernumber = :rid "
+            . "and biblionumber = :bid";
```

Comment:
I think the main use case of concern is the one that @EreMaijala brought up on the mailing list: a bib record representing a multi-volume set, with items for different volumes. In this (fairly common) scenario, a user may indeed wish to place holds on more than one item.